### ISSUE TITLE
Why loss is not decreased in variational autoencoder?

### ISSUE THREAD

**Issue by JasOlean** [keras-team/keras#12513]
autoencoder.compile(optimizer= optimizer, loss = vae_loss, metrics = ['acc'])

def vae_loss(x, x_decoded_mean):
    xent_loss = original_dim * metrics.mse(x, x_decoded_mean)
    kl_loss = - 0.5 * K.sum(1 + z_log_var - K.square(z_mean) - K.exp(z_log_var), axis = -1)
    loss = K.mean(xent_loss + kl_loss)
    return loss

Epoch 1/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.6721 - acc: 0.3227 - val_loss: 4.6551 - val_acc: 0.3275
Epoch 2/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6553 - val_acc: 0.3275
Epoch 3/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5684 - acc: 0.3287 - val_loss: 4.6560 - val_acc: 0.3275
Epoch 4/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5682 - acc: 0.3287 - val_loss: 4.6550 - val_acc: 0.3275
Epoch 5/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6564 - val_acc: 0.3275
Epoch 6/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6567 - val_acc: 0.3275
Epoch 7/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6545 - val_acc: 0.3275
Epoch 8/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5682 - acc: 0.3287 - val_loss: 4.6545 - val_acc: 0.3275
Epoch 9/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6543 - val_acc: 0.3275
Epoch 10/15
265199/265199 [==============================] - 12s 43us/step - loss: 4.5682 - acc: 0.3287 - val_loss: 4.6550 - val_acc: 0.3275
Epoch 11/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.5682 - acc: 0.3287 - val_loss: 4.6558 - val_acc: 0.3275
Epoch 12/15
265199/265199 [==============================] - 11s 41us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6550 - val_acc: 0.3275
Epoch 13/15
265199/265199 [==============================] - 11s 43us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6557 - val_acc: 0.3275
Epoch 14/15
265199/265199 [==============================] - 12s 43us/step - loss: 4.5682 - acc: 0.3287 - val_loss: 4.6549 - val_acc: 0.3275
Epoch 15/15
265199/265199 [==============================] - 12s 44us/step - loss: 4.5683 - acc: 0.3287 - val_loss: 4.6551 - val_acc: 0.3275

Moreover, how can I know xent_loss and kl_loss vaules without tensor zip?

**Comment by ymodak** [comment]
Please provide the system information asked below. Thanks!
System information:
Have I written custom code (as opposed to using example directory):
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
TensorFlow backend (yes / no):
TensorFlow version:
Keras version:
Python version:
CUDA/cuDNN version:
GPU model and memory:
You can obtain the TensorFlow version with:
python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
You can obtain the Keras version with:
python -c 'import keras as k; print(k.__version__)' 
Describe the problem:
Describe the problem clearly here. Be sure to convey here why it's a bug in Keras or a feature request.
Source code / logs:
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.
